intro-to-R-tidyverse/README.md

```diff
@@ -0,0 +1,15 @@
+# Intro to R and Tidyverse Training Module
+
+This CCDL-designed module covers the introduction to R and the Tidyverse package. It is designed to be taught in 1.5 hours. 
+
+It goes through the basics of R _(including data types, data structures, and good scientific coding practices)_ and the tidyverse package _(including dplyr pipes, common tidyverse functions, and the usage of ggplot2 for data visualization)_, thus providing the tools needed to move onto the subsequent training modules. 
```

Comment:
Can you note the Docker container in here, mirroring: https://github.com/AlexsLemonade/training-modules/tree/b92055b3292b38a1bf42e363d32e33b097f25541/RNA-seq